sim/src/make/generator.rs

```diff
@@ -428,6 +428,56 @@ impl ScenarioGenerator {
         let mut s = Scenario::empty(map, "random people going to/from work");
         s.only_seed_buses = None;
         timer.start_iter("create people", total_ppl);
+
+        let incoming_connections = map.all_incoming_borders();
+        let outgoing_connections = map.all_outgoing_borders();
+        // transit
+        for (_home, _num_ppl) in &residences {
+            // TODO it would be nice to weigh border points by for example lane count
+            let random_incoming_border = incoming_connections.choose(rng).unwrap();
+            let random_outgoing_border = outgoing_connections.choose(rng).unwrap();
+            let b_random_incoming_border = incoming_connections.choose(rng).unwrap();
+            let b_random_outgoing_border = outgoing_connections.choose(rng).unwrap();
+            let mode = TripMode::Drive; // TODO: more interesting mode selection, requires refactoring of mode selection function
+            let test_mode = select_trip_mode(Distance::meters(20.0 * 1000.0));
```

Comment:
I don't see this function? Was this tmp test code?

Reply:
Ops, danger of having some code uncommitted and testing anyway. I will fix it soon.

Reply:
Should be now fixed (is it OK to force-push in PRs?)

Reply:
Looks fine to me. Force pushing is fine; I don't understand git well enough to know how to deal cleanly with rebasing in the middle of a PR. Probably time to learn that...

This PR looks ready to me; can we merge it, or were you doing more things in it? (Also, if you'd ever prefer to coordinate by chat, https://join.slack.com/t/abstreet/shared_invite/zt-fr35wtef-BWIJpx0vlvBOVGvHET9XNA is the A/B Street slack, or I'm also on osmus)

Reply:
> This PR looks ready to me; can we merge it

Yes - it is not thoroughly tested (see this issue) and see also other issues noted above.

I plan to fix at least what was mentioned above, but feel free to merge it and tweak it on your own.

And also improve this topic further (#154) but it may be today, in one week or in one year.

Reply:
> is the A/B Street slack, or I'm also on osmus

joined to A/B one, though I am not expecting to be active. I am appearing relatively frequently on osmus (though no promises about that). If you ever want to me reach me - see method mentioned in PM on AB slack.